sdk/eventhub/event-hubs/src/managementClient.ts

```diff
@@ -132,6 +133,26 @@ export class ManagementClient extends LinkEntity {
     this.entityPath = context.config.entityPath as string;
   }
 
+  /**
+   * Gets the security token for the management application properties.
+   * @ignore
+   * @internal
+   */
+  async getSecurityToken() {
+    if (this._context.tokenCredential instanceof SharedKeyCredential) {
+      // the security_token has the $management address removed from the end of the audience
+      const audienceParts = this.audience.split("/");
```

Comment:
If it's possible it's helpful to add a comment showing what the string you're parsing apart looks like.